### ISSUE TITLE
Exception TypeError in MethodView example

### ISSUE THREAD

**Issue by fapsi** [flask-admin/flask-admin#2289]
Can someone reproduce this error while running the example and clicking on any api-link?
```
Traceback (most recent call last):
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask/app.py", line 2548, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.handle_exception(e)
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask_admin/base.py", line 69, in inner
    return self._run_view(f, *args, **kwargs)
  File "/home/fapsi/Projects/flask-admin/examples/methodview/venv/lib/python3.10/site-packages/flask_admin/base.py", line 368, in _run_view
    return fn(self, *args, **kwargs)
TypeError: View.as_view.<locals>.view() takes 0 positional arguments but 1 was given
```

**Comment by jayaddison** [comment]
This seems to be due to an incompatibility with `Flask>=2.2.0` - in particular, `flask-admin`'s [`@expose_plugview` decorator](https://github.com/flask-admin/flask-admin/blob/b22e2745568c5e14d85abfff40fc86ba04ae3656/flask_admin/base.py#L32-L50) seems to have issues when combined with the changes from pallets/flask#4624.

A short-term workaround is to pin to `Flask<2.2.0`.

As visible in the stacktrace provided, the problem appears at runtime during the [`_run_view` method](https://github.com/flask-admin/flask-admin/blob/b22e2745568c5e14d85abfff40fc86ba04ae3656/flask_admin/base.py#L69) in the view function wrapper.

**Comment by jayaddison** [comment]
Resolved by #2316, I think (pending release).

**Comment by fapsi** [comment]
Yes, fixed in 1.6.1 in my tests. Thanks again!
